Src/IronPython/Modules/Builtin.cs

```diff
@@ -925,18 +934,23 @@ public class mapobject : IEnumerable {
         }
 
         public static object max(CodeContext/*!*/ context, object x, object y, [ParamDictionary] IDictionary<object, object> dict) {
-            object method = GetMaxKwArg(dict);
+            var kwargTuple = GetMaxKwArg(dict);
+            object method = kwargTuple.Item1;
```

Comment:
Not something I'd block the PR on, but `GetMaxKwArg` could use an argument to check to make sure default is allowed, and if not throw:
> TypeError: Cannot specify a default for max() with multiple positional arguments